spec/factories/application_drafts.rb

```diff
@@ -8,11 +8,12 @@
       association :project1, :accepted, factory: :project
       plan_project1         { FFaker::Lorem.paragraph }
       heard_about_it        { [FFaker::Lorem.paragraph] }
+      work_weeks            { [FFaker::Lorem.paragraph] }
       why_selected_project1 { FFaker::Lorem.paragraph }
       working_together      { FFaker::Lorem.paragraph }
       misc_info             { FFaker::Lorem.paragraph } # NOTE not a required field
 
-      after(:create) do |draft|
+      after(:create) do |draft, evaluator|
```

Comment:
Hm, I don't get this change I'm afraid 🙈

Reply:
That is a leftover from older commits, thank you for finding it! 😱